erizo_controller/erizoClient/src/webrtc-stacks/ChromeStableStack.js

```diff
@@ -39,6 +40,88 @@ Erizo.ChromeStableStack = function (spec) {
 
     that.peerConnection = new WebkitRTCPeerConnection(that.pcConfig, that.con);
 
+    var addSim = function (spatialLayers) {
+      var line = 'a=ssrc-group:SIM';
+      spatialLayers.forEach(function(spatialLayerId) {
+        line += ' ' + spatialLayerId;
+      });
+      return line + '\r\n';
+    };
+
+    var addGroup = function(spatialLayerId, spatialLayerIdRtx) {
+      return 'a=ssrc-group:FID ' + spatialLayerId + ' ' + spatialLayerIdRtx + '\r\n';
+    };
+
+    var addSpatialLayer = function (cname, msid, mslabel, label, spatialLayerId, spatialLayerIdRtx) {
+      return  'a=ssrc:' + spatialLayerId + ' cname:' + cname +'\r\n' +
+              'a=ssrc:' + spatialLayerId + ' msid:' + msid + '\r\n' +
+              'a=ssrc:' + spatialLayerId + ' mslabel:' + mslabel + '\r\n' +
+              'a=ssrc:' + spatialLayerId + ' label:' + label + '\r\n' +
+              'a=ssrc:' + spatialLayerIdRtx + ' cname:' + cname +'\r\n' +
+              'a=ssrc:' + spatialLayerIdRtx + ' msid:' + msid + '\r\n' +
+              'a=ssrc:' + spatialLayerIdRtx + ' mslabel:' + mslabel + '\r\n' +
+              'a=ssrc:' + spatialLayerIdRtx + ' label:' + label + '\r\n';
+    };
+
+    var enableSimulcast = function (sdp) {
+      var r, a, b;
```

Comment:
Consider using more descriptive variable names. I know it's still a WIP but we have to start someday.